src/main/cpp/profiler/spark_rapids_profile_converter.cpp

```diff
@@ -740,6 +1145,33 @@ int main(int argc, char* argv[])
       } else {
         convert_to_nvtxt(in, std::cout, opts);
       }
+    } else if (opts.nvtxw) {
+      if (opts.output_path) {
+        void* nvtxwModuleHandle              = nullptr;
+        nvtxwInterfaceCore_t* nvtxwInterface = nullptr;
+        nvtxwSessionHandle_t session;
+        nvtxwStreamHandle_t stream;
+        errorCode = initialize_nvtxw(in,
```

Comment:
This is only used once thus can be declared and used immediately:
```suggestion
        auto const errorCode = initialize_nvtxw(in,
```